pkg/query-service/app/http_handler.go

```diff
@@ -2192,6 +2210,136 @@ func (aH *APIHandler) WriteJSON(w http.ResponseWriter, r *http.Request, response
 	w.Write(resp)
 }
 
+// Preferences
+
+func (ah *APIHandler) getUserPreference(
+	w http.ResponseWriter, r *http.Request,
+) {
+	preferenceId := mux.Vars(r)["preferenceId"]
+	user := common.GetUserFromContext(r.Context())
+
```

Comment:
if the orgId is not present that means the user is consuming the older JWT token hence we return 401 from here indicating the frontend to explicit logout from here